src/lightning_app/utilities/cluster_logs.py

```diff
@@ -39,35 +39,47 @@ def _push_log_events_to_read_queue_callback(read_queue: queue.PriorityQueue):
     Returns callback function used with `on_message_callback` of websocket.WebSocketApp.
     """
 
-    def callback(ws_app: WebSocketApp, msg: str):
-        # We strongly trust that the contract on API will hold atm :D
-        event_dict = json.loads(msg)
-        labels = _ClusterLogEventLabels(**event_dict["labels"])
-
-        if "message" in event_dict:
-            message = event_dict["message"]
-            timestamp = dateutil.parser.isoparse(event_dict["timestamp"])
-            event = _ClusterLogEvent(
-                message=message,
-                timestamp=timestamp,
-                labels=labels,
-            )
-            read_queue.put(event)
+    def callback(_: WebSocketApp, msg: str):
+        for ev in _parse_log_event(msg):
+            read_queue.put(ev)
 
     return callback
 
 
+def _parse_log_event(msg: str) -> List[_ClusterLogEvent]:
+    # We strongly trust that the contract on API will hold atm :D
+    event_dict = json.loads(msg)
+    log_entry_labels = {}
+    allowed_log_entry_labels = {x.name for x in dataclasses.fields(_ClusterLogEventLabels)}
```

Comment:
I think, since we don't use most of the labels, now we can:
-  clean up fields in `_ClusterLogEventLabels` - it seems we use only `labels.level`?

or:
- get rid of `_ClusterLogEventLabels` class at all and simplify the things, define labels attribute of `_ClusterLogEvent` class as `field` (but in this case we lose opportunity to access to the log labels as an attribute via `.` (`log_entry.labels.level`), we should use `log_entry.labels['level']`):

```
from dataclasses import dataclass, field

@dataclass
class _ClusterLogEvent(_OrderedLogEntry):
    labels: field(default_factory=dict)
```

Reply:
See PR changes, we can still access things via `.` and have type annotations for common things :)